examples/b2c/checkout/lwr-order-confirmation-email-flow/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
+        <members>OrderSummary.OCE_Account_First_Name__c</members>
+        <members>OrderSummary.OCE_Account_Last_Name__c</members>
+        <members>OrderSummary.OCE_Order_Items__c</members>
+        <members>OrderSummary.OCE_Order_Url__c</members>
+        <members>OrderSummary.OCE_Order_Total__c</members>
+        <members>OrderSummary.OCE_Payment_Method__c</members>
+        <members>OrderSummary.OCE_Shipping_Address__c</members>
+        <members>OrderSummary.OCE_Shipping_Method__c</members>
+        <members>OrderSummary.OCE_Order_Url__c</members>
+        <name>CustomField</name>
+    </types>
+    <types>
+        <members>unfiled$public/B2B2C_Order_Confirmation_Email</members>
+        <name>EmailTemplate</name>
+    </types>
+    <types>
+        <members>B2B2C_Order_Confirmation_Email_Notification_Flow</members>
+        <name>Flow</name>
+    </types>
+    <types>
+        <members>OrderSummary.B2B2C_Order_Confirmation_Email_Alert</members>
+        <name>WorkflowAlert</name>
+    </types>
+    <version>53.0</version>
```

Comment:
Is the api version value right?  Is this feature introduced at 234 build?

Reply:
@tarcang , I have modified it to 59 the current version. Thanks